src/main/java/gitflow/ui/GitflowWidget.java

```diff
@@ -163,6 +203,8 @@ private void update() {
             emptyTextAndTooltip();
             return;
         }
+        checkSupportedVersion();
```

Comment:
This is problematic and might create performance issues if you're running this on EVERY git update